packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -382,6 +450,40 @@ export class Pipeline extends cdk.Construct implements cpapi.IPipeline {
   }
 
   private renderStages(): cloudformation.PipelineResource.StageDeclarationProperty[] {
+    // handle cross-region CodePipeline overrides here
+    let crossRegion = false;
+    this.stages.forEach((stage, i) => {
+      stage.actions.forEach((action, j) => {
+        if (action.region) {
+          crossRegion = true;
+          this.pipelineResource.addPropertyOverride(`Stages.${i}.Actions.${j}.Region`, action.region);
```

Comment:
Why do we need a property override here? Wouldn't it make more sense that `Action` will just render a `Region` field?

Reply:
We need it, as in the current schema `Action` does not have a field `region`.